src/templates/resources/footer.json

```diff
@@ -1284,121 +1375,118 @@
         "buttonClose": "Chiudi",
         "links": [
             {
-                "url": "/help",
-                "text": "Aiuto?"
-            },
-            {
-                "url": "/account/login",
-                "text": "Accedi",
-                "rel": "nofollow"
+                "url": "/account/login/",
+                "text": "Accedi"
             },
             {
-                "url": "/account/register",
+                "url": "/account/register/",
                 "text": "Registrati"
             },
             {
-                "url": "/blog",
+                "url": "/blog/",
                 "text": "Blog"
             },
             {
-                "url": "/apps",
-                "text": "App"
+                "url": "/account/info/",
+                "text": "Il mio account"
+            },
+            {
+                "url": "/help/",
+                "text": "Aiuto?"
             }
         ],
         "cuisines": "Cucine",
         "cuisineLinks": [
             {
-                "url": "/domicilio/pizza",
+                "url": "/domicilio/pizza/",
                 "text": "Pizza a domicilio"
             },
             {
-                "url": "/domicilio/cinese",
+                "url": "/domicilio/cines/e",
                 "text": "Cinese a domicilio"
             },
             {
-                "url": "/domicilio/sushi",
+                "url": "/domicilio/sushi/",
```

Comment:
I also tried this link and it doesn't work, but this one does: `/domicilio/vicino-a-me/sushi-giapponese/`